utility/test/transaction_test.go

```diff
@@ -86,18 +86,18 @@ func TestUtilityContext_GetSignerCandidates(t *testing.T) {
 	test_artifacts.CleanupTest(ctx)
 }
 
-func TestUtilityContext_GetTransactionsForProposal(t *testing.T) {
+func TestUtilityContext_CreateAndApplyBlock(t *testing.T) {
 	ctx := NewTestingUtilityContext(t, 0)
 	tx, _, _, _ := newTestingTransaction(t, ctx)
 	proposer := getAllTestingValidators(t, ctx)[0]
 	txBz, err := tx.Bytes()
 	require.NoError(t, err)
 	require.NoError(t, ctx.CheckTransaction(txBz))
-	txs, txResults, er := ctx.GetProposalTransactions([]byte(proposer.GetAddress()), 10000, nil)
+	appHash, txs, er := ctx.CreateAndApplyProposalBlock([]byte(proposer.GetAddress()), 10000)
 	require.NoError(t, er)
 	require.Equal(t, len(txs), 1)
 	require.Equal(t, txs[0], txBz)
-	requireValidTestingTxResults(t, tx, txResults)
+	require.NotNilf(t, appHash, "")
```

Comment:
I think you can switch `NotNilf` to `NotEmpty` to make it more idiomatic: https://pkg.go.dev/github.com/stretchr/testify/assert#NotEmpty.

<img width="1248" alt="Screen Shot 2022-11-03 at 10 37 58 AM" src="https://user-images.githubusercontent.com/1892194/199794700-1147ca37-6f49-4844-a772-889716bb0fed.png">